README_DataTeam.md

```diff
@@ -0,0 +1,17 @@
+# Guide for the Data Team
+
+This is a short Readme specific to Data Team (MSFC) tasks, and usage of Zenhub.
+
+## Zenhub
+
+To make full use of Zenhub you need to install the browser extension. This will allow you to the Workspaces under the Zenhub tab in each repo.
```

Comment:
Should we make a note that the extension is only available in certain browsers - I think you mentioned it's only available for Firefox and Chrome.

Reply:
https://help.zenhub.com/support/solutions/articles/43000507578-installing-the-zenhub-extension-for-cloud